### PR TITLE
Career Quiz Result UI 

### PR DESCRIPTION
**Pull request by LuckyHariharan**
In this PR I built the Result UI for the career quiz. 
Key Notes:
- Button is currently using default blue, by passing down "blue" as a prop.
- The component is 'dumb' so it will always produce just software engineer.
- The border is hardcoded as rounded-[40px] because none of the default options worked, I think because of the image resolution.

Next Steps:
- Route pages

Current result, regardless of user input:
image
Figma design:
image

### CODE REVIEW

File: pages/studentPortal/quizcomponent/career-results.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Button } from "../../../components/ui/Button";
+import QuizNavbar from "./quiznavbar";
+
+const Career_Results = () => {
+  return (
+    <div className="grid  w-full mx-auto text-center justify-items-center">
+      <QuizNavbar></QuizNavbar>
```
Comment (vithushan19) [inline pages/studentPortal/quizcomponent/career-results.tsx:8]: Couldn't this be `<QuizNavbar />`

File: pages/studentPortal/quizcomponent/career-results.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Button } from "../../../components/ui/Button";
+import QuizNavbar from "./quiznavbar";
+
+const Career_Results = () => {
```
Comment (vithushan19) [inline pages/studentPortal/quizcomponent/career-results.tsx:5]: should be `CareerResults`

File: pages/studentPortal/quizcomponent/career-results.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      <QuizNavbar></QuizNavbar>
+      <div className="font-bold mt-4 text-2xl ">YOUR RESULTS</div>
+      <div className=" text-lg font-semibolds ">
+        Here are jobs in tech you’re compatible with. Click on a career to learn
+        more!
+      </div>
+      <img
+        src="/images/software-engineer.png"
+        alt="Software Engineer"
+        className=" p-4 rounded-[40px]  md:h-60 md:w-80 "
```
Comment (vithushan19) [inline pages/studentPortal/quizcomponent/career-results.tsx:17]: i'm ok with the hardcoded 40px but did you try using `object-fit` as well in addition to `rounded-lg`? If this works, im fine with this.

**Review by vithushan19 — CHANGES_REQUESTED** [review]
looks pretty good, just have some naming/organization suggestions

File: pages/studentPortal/quizcomponent/career-results.tsx (hunk 1 above)
Comment (LuckyHariharan) [reply pages/studentPortal/quizcomponent/career-results.tsx:8]: Yup! ty for pointing this out, fixed.

File: pages/studentPortal/quizcomponent/career-results.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  return (
+    <div className="grid  w-full mx-auto text-center justify-items-center">
+      <QuizNavbar></QuizNavbar>
+      <div className="font-bold mt-4 text-2xl ">YOUR RESULTS</div>
+      <div className=" text-lg font-semibolds ">
+        Here are jobs in tech you’re compatible with. Click on a career to learn
+        more!
+      </div>
+      <img
+        src="/images/software-engineer.png"
```
[the remark this answers is not in the document]
Comment (LuckyHariharan) [reply pages/studentPortal/quizcomponent/career-results.tsx]: absolutely, fixed.

**Pull request merged**